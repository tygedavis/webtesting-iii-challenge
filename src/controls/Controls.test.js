import React from 'react';
import { render } from '@testing-library/react';

import Controls from './Controls';

// test('Toggles closed and locked states', () => {
//   const locked = true;
//   const closed = true;
//   const toggleLocked = true;
//   const toggleClosed = true;
//   const tree = render(<Controls 
//     locked = {locked}
//     closed = {closed}
//     toggleLocked = {toggleLocked}
//     toggleClosed = {toggleClosed}
//     />)
//   expect(tree.baseElement).toMatchSnapshot()
// })

describe('<Controls />', () => {
  it('Renders Controls', () =>{
    render(<Controls />)
  })
})