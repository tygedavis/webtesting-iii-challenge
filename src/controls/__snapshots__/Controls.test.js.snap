// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toggles closed and locked states 1`] = `
<body>
  <div>
    <div
      class="controls panel"
    >
      <button
        class="toggle-btn"
      >
        Unlock Gate
      </button>
      <button
        class="toggle-btn"
        disabled=""
      >
        Open Gate
      </button>
    </div>
  </div>
</body>
`;
